cloud-init-configs/ubuntu/spin-up-jammy.sh

```diff
@@ -5,17 +5,17 @@ if (find user-data) then
   echo "user-data exists"
 else
   echo "Creating user-data"
-  touch user-data
-  cat << EOF > user-data
+  sshkey="$(cat ~/.ssh/id_ed25519.pub)"
```

Comment:
Why are you setting a varialbe? Could you just put the subslhell on line 18?

Reply:
I must have mistyped when I originally tried putting the subshell in the tee call. I didn't see an error so thought there was a collision between heredoc and shell commands. fixed now